### PR TITLE
Add jira ticket instructions to future migrations

### PR DESCRIPTION
**Pull request by MGibson1**
## Objective

Adds process instructions to try and keep track of future migrations a little better.

### CODE REVIEW

File: docs/contributing/database-migrations/edd.mdx [hunk 1]
```diff
@@ -190,6 +190,13 @@ Create a PR moving the future scripts.
 - `DbScripts_future` to `DbScripts`, prefix the script with the current date, but retain the
   existing date.
 - `dbo_future` to `dbo`.
+  <bitwarden>
+    <li>
+      Create a ticket in Jira with a `Due Date` of the release date to ensure future migrations are
+      merged in and ready to be executed. Set the ticket which created the future migration as a
```
Comment (withinfocus) [inline docs/contributing/database-migrations/edd.mdx:196]: ⛏️ ticket "that".

**Review by withinfocus — APPROVED** [review]

File: docs/contributing/database-migrations/edd.mdx (hunk 1 above)
Comment (eliykat) [reply docs/contributing/database-migrations/edd.mdx:196]: As a native speaker, that truly is a ⛏️  😂  but I don't disagree

File: docs/contributing/database-migrations/edd.mdx [hunk 2]
```diff
@@ -190,6 +190,13 @@ Create a PR moving the future scripts.
 - `DbScripts_future` to `DbScripts`, prefix the script with the current date, but retain the
   existing date.
 - `dbo_future` to `dbo`.
+  <bitwarden>
+    <li>
```
Comment (Hinton) [inline docs/contributing/database-migrations/edd.mdx:194]: I assume we need the `li` here and `-` doesn't work?

File: docs/contributing/database-migrations/edd.mdx (hunk 1 above)
Comment (MGibson1) [reply docs/contributing/database-migrations/edd.mdx:196]: blame the 🤖

File: docs/contributing/database-migrations/edd.mdx (hunk 2 above)
Comment (MGibson1) [reply docs/contributing/database-migrations/edd.mdx:194]: yeah... not super happy with it but the `\n`s needed messed up the markdown spacing or failed to compile due to trying to `</li>` in the middle of the `<bitwarden>` element 😭

**Pull request merged**